pkg/apis/configuration/validation/validation.go

```diff
@@ -184,15 +188,49 @@ func validateStatusMatch(s string, fieldPath *field.Path) field.ErrorList {
 	return allErrs
 }
 
-func validateStatusCodeInRange(status string) string {
-	code, err := strconv.ParseInt(status, 10, 64)
+func validateStatusCodeRange(statusRange []string) string {
```

Comment:
this function always uses  2 elements of the  statusRange slice. Because of that, I suggest changing the signature to take 2 arguments. For example, `validateStatusCodeRange(min sting, max string)`. This will make the code easier to understand.

Alternatively, it is possible to bring the following the code into this function and make it take a string argument like `validateStatusCodeRange(statusRange string)`
```
statusRange := strings.Split(value, "-")
if len(statusRange) != 2 {
				allErrs = append(allErrs, field.Invalid(fieldPath, s, "ranges must only have 2 numbers"))
			}
```
Let me know what you think
```